app/assets/stylesheets/posts.scss

```diff
@@ -90,7 +90,7 @@
 	.gm-style div div div {
 		width: 100%;
 		text-align: left;
-		max-width: 200px;
+		max-width: 400px;
```

Comment:
What's the reasoning behind changing this? This looks kind of weird when there is barely any text in the bubble.